cmake/multi_image.cmake

```diff
@@ -184,6 +189,7 @@ function(add_child_image_from_source)
              # Child image always uses the same revision as parent board.
              BOARD_REVISION ${BOARD_REVISION}
              KCONF ${ACI_NAME}_CONF_FILE
+	     DTS ${ACI_NAME}_DTC_OVERLAY_FILE
```

Comment:
nit: indentation seems off (at least in GH)

Reply:
There was tab instead of white spaces. Corrected.